tests/js/t17177.nim

```diff
@@ -0,0 +1,10 @@
+import std/asyncjs
+
```

Comment:
I will move this to `tasyncjs` in followup work after https://github.com/nim-lang/Nim/pull/17189

Reply:
I agree